test/acceptance/tests/basic/terraform/basic-install/main.tf

```diff
@@ -64,7 +64,8 @@ variable "launch_type" {
 variable "consul_ecs_image" {
   description = "Consul ECS image to use."
   type        = string
-  default     = "docker.mirror.hashicorp.services/hashicorpdev/consul-ecs:latest"
+  // TODO: default     = "docker.mirror.hashicorp.services/hashicorpdev/consul-ecs:latest"
```

Comment:
Need to change this back to `:latest` once https://github.com/hashicorp/consul-ecs/pull/135 is merged.